doc/news/changes/minor/20190708LuelEmishaw

```diff
@@ -0,0 +1 @@
+I described [In] and [out] parameters of flatten_triangulation()
```

Comment:
Take a look at the other files in this directory. They're all pretty formulaic in their format. See if you can imitate it.